data/energy_of_e/energy_of_e.py

```diff
@@ -0,0 +1,10 @@
+
+import math
+Rh = 13.60648962
+n = 3
+energy_true = (-Rh/(n**2))
+energy_nlcc = energy_of_e(n)
+
+
+#result = math.isclose(energy_true, energy_nlcc, rel_tol =1e-3)
```

Comment:
@SamCox822 i think you accidentally commented out the result line.

Reply:
Fixed it!